sources/packages/web/src/components/common/modals/EducationProgramDeactivationModal.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <v-form ref="deactivateProgramForm">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      title="Confirm deactivation"
+      ref="confirmDeactivationModal"
+      okLabel="Confirm deactivation"
```

Comment:
I am confused here on 2 things. the `okLabel` property belongs to `ConfirmModal` and not the `ModalDialogBase` right?? 

and I do not see that `keepModalOpen` is a property to the component, but an option to the resolvePromise method.

Let me know if I am missing something here.

Reply:
Properties removed.